app/controllers/characteristics_controller.rb

```diff
@@ -1,6 +1,26 @@
 class CharacteristicsController < ApplicationController
   load_and_authorize_resource
 
+  def new
+    render layout: "custom"
+  end
+
+  def create
+    @characteristic = Characteristic.new(params[:characteristic])
+    # @design_method.principle = ""
```

Comment:
Remove stray commented out code, this applies to the entire branch. Scroll through this diff and clean them all up. Thanks!